Assets/Scripts/WorldEngine/Cultures/Preferences/CellCulturalPreference.cs

```diff
@@ -16,7 +16,8 @@ public class CellCulturalPreference : CulturalPreference
     private float _newValue;
 
 #if DEBUG
-    private long _lastUpdateDate = Manager.CurrentWorld.CurrentDate;
+    // Null ref when loading from file
```

Comment:
Create an issue for this instead of leaving it commented

Reply:
see https://github.com/Zematus/Worlds/issues/350